scripts/_lib.sh

```diff
@@ -79,6 +79,20 @@ function build_container {
         $container
 }
 
+# build the container and tag it as local
+function build_local {
+    local container=$1
+    local APP_REVISION=`git rev-parse --abbrev-ref HEAD`
+    local DC="docker-compose -f docker-compose.yml -f docker-compose-connect.yml -f docker-compose-test.yml"
+
+    echo_message "Building local container $container"
+    $DC build \
+        --build-arg GIT_REVISION=$APP_REVISION \
+        --build-arg VERSION=local \
+        $container
+    docker tag aether-$container:latest aether-$container:local
+}
+
```

Comment:
Wouldn't it be easier to change the current `build_container` function instead of creating a new one?

```bash
function build_container {
    local container=$1
    local APP_REVISION=`git rev-parse --abbrev-ref HEAD`
    if [ ! -f VERSION ]; then
        local APP_VERSION="0.0.0"
    else
        local APP_VERSION=`cat ./VERSION`
    fi
    # use the VERSION file or 0.0.0. in develop branch if missing second argument
    local TAG=${2:-$APP_VERSION}

    local DC="docker-compose -f docker-compose.yml -f docker-compose-connect.yml -f docker-compose-test.yml"

    echo_message "Building container aether-$container:$TAG"
    $DC build \
        --no-cache --force-rm --pull \
        --build-arg GIT_REVISION=$APP_REVISION \
        --build-arg VERSION=$APP_VERSION \
        $container

    docker tag aether-$container:latest aether-$container:$TAG
}
```